providers/gov/usgs/normalizer.py

```diff
@@ -0,0 +1,82 @@
+from share.normalize import *
+from share.normalize.utils import format_doi_as_url
+
+
+class Person(Parser):
+    given_name = Maybe(ctx, 'given')
+    family_name = ctx.family
+
+
+class Contributor(Parser):
+    order_cited = ctx('index')
+    cited_name = Join(Concat(Maybe(ctx, 'given'), ctx.family), joiner=' ')
+    person = Delegate(Person, ctx)
+
+
+class Link(Parser):
+    url = ctx
+    type = RunPython('get_link_type', ctx)
+
+    def get_link_type(self, link):
+        if 'dx.doi.org' in link:
+            return 'doi'
+        elif 'usgs.gov' in link:
```

Comment:
If you have `return`s in your `if` statements don't use `else` or `elif`

Excerpt from [Effective Go](https://golang.org/doc/effective_go.html#if)

> In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.